libfdo-data/src/test_common.rs

```diff
@@ -34,6 +34,27 @@ pub fn run_external(script: &str, args: &[&str]) -> Output {
         .join("test_scripts")
         .join(format!("{}.go", script));
 
+    match std::fs::remove_file(format!(
+        "{}/target/debug/libfdo_data.so.0",
+        std::env::var("PWD").unwrap()
+    )) {
+        Ok(_) => (),
+        Err(e) if e.kind() == std::io::ErrorKind::NotFound => (),
+        Err(e) => panic!("Failed to remove libfdo_data.so symlink {:?}", e)
+
+    }
+    std::os::unix::fs::symlink(
+        format!(
+            "{}/target/debug/libfdo_data.so",
+            std::env::var("PWD").unwrap()
```

Comment:
So, I know how ugly it can be to find things, but maybe let's use `env!("CARGO_MANIFEST_DIR")`, since `PWD` feels very fragile?

Reply:
done